workflow/rules/common.smk

```diff
@@ -539,17 +597,22 @@ def get_selected_annotations():
 
 
 def get_annotated_bcf(wildcards):
```

Comment:
We could add `datatype = "variants" if wildcards.analysis == "variants" else "arriba"`.
While it would be a bit unintuitive replacing `fusions` as the default datazype by the caller `arriba` this would skip calling `bcftools concat` where `delly` and `freebayes` calls are concatenated for variant calling.
For fusion calls there is only one caller and this step could be skipped at the expanse of a bit confusing datatype assignment.

Reply:
not a fan of that. I'd rather skip it in a different way.